api/system/v1alpha1/zone.proto

```diff
@@ -6,4 +6,8 @@ option go_package = "github.com/kumahq/kuma/api/system/v1alpha1";
 
 // Zone defines the Zone configuration used at the Global Control Plane
 // within a distributed deployment
-message Zone {}
+message Zone {
+  // enable allows to turn the zone on/off and exclude the whole zone from
+  // balancing traffic on it
+  bool enabled = 1;
```

Comment:
that's a breaking change. We could do a BoolWrapper and treat `nil` as true

Reply:
Agree, we should!

Reply:
Add a card that for 1.1 we should change this to true by default

Reply:
Do you mean treat `nil` as false since 1.1?

Reply:
yes